install/clax_install.sh

```diff
@@ -91,6 +92,9 @@ function change_owner {
 
 function platform_config {
     cat >$dir/enable_clax_inetd_generic.sh <<GEN
+#!/bin/ksh
```

Comment:
Because I thought it would be good idea to use the same one used in the main install script. If you reach this point (executing the .sh generated by the install script), it's because you have ksh.